infra/main.tf

```diff
@@ -318,17 +318,18 @@ data "http" "load_balancer_warm_up" {
 ### Firestore ###
 
 # The following checks Asset Inventory for an existing Firestore database
-data "google_cloud_asset_resources_search_all" "firestore_database" {
+data "google_cloud_asset_resources_search_all" "default_firestore_database" {
   provider = google-beta
   scope    = "projects/${var.project_id}"
+  query    = "displayName:(default)"
```

Comment:
This change makes sense to me, especially with the change of allowing multiple firestore databases. I would want to confirm that this change successfully allows multiple deployments of the same JSS to the one project (and that they are able to operate independently).

Reply:
+1 

I've confirmed that this change successfully allows multiple deployments in 1 project